### PR TITLE
Global Web Development Summit Capstone Project

### PR DESCRIPTION
**Pull request by mrbhi**
- Added the summit date and venue
- Added list of programs/events for the summit
- Added list of speakers for the summit
- Added various partners for the summit
- Added the logo design during a conference on the about page
- Added the toggle menu functionality for the mobile menu
- Added scroll-to-top functionality to my project

**Please kindly find the link to my zoom video giving a brief description of my capstone project below.**
Thanks for your PR

https://www.loom.com/share/a957a1c646f243eb90c2a8788e35493e

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+			<ul class="menu-list">
+				<div class="close">
+						<img class="hamburger-menu-close" src="./img/hamgurger-close-black.png" width="40px" alt="Hamburger-Menu-Close" />
+				</div>
+				<li class="menu-item"><a href="about.html" class="menu-link">About</a></li>
+				<li class="menu-item"><a href="#" class="menu-link">Program</a></li>
+				<li class="menu-item"><a href="#" class="menu-link">Join</a></li>
+				<li class="menu-item"><a href="#" class="menu-link">Sponsor</a></li>
+				<li class="menu-item"><a href="#" class="menu-link">News</a></li>
+				<li class="menu-item campaign1"><a href="#" class="menu-link campaign">CC Campaign</a></li>
```
Comment (aimalamiri) [inline index.html:39]: Good job! but as you know it's part of the core requirements to follow the original design idea, so please make sure to update your nav link colors, sizing, and spacing to match the original design.

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+				<p class="headline-lead">
+					The three days of the conference will contribute to build long-term
+					profitable and sustainable communication among the participants in a
+					wide variety of Informationa Technology areas with the common interest in Applied
+					Science, Engineering and Technology.
+				</p>
+			</div>
+			<h2 class="date">2023.03.23(TUE) ~ 25(WED)</h2>
+			<p class="venue">@ECO-HOTEL & SUIT, LAGOS, NIGERIA</p>
+		</div>
```
Comment (aimalamiri) [inline index.html]: It looks great, but you can improved its design by following the original design as much as possible. So please try to update your background image (I can see a blue picture in your video, but it's not available in the one you deployed), header text sizing and colors.

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+	</section>
+
+	<section class="main-program">
+		<div class="main-event-details">
+			<h2>Main Program</h2>
+			<ul class="program">
```
Comment (aimalamiri) [inline index.html]: It looks fantastic, but it's highly recommended to use the same background color (image) for your own design, so please try to update this part of your page as well.

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+	<section class="speakers">
+		<h2 class="member">Team Members</h2>
+		<div class="grid flex">
+			<!-- <div class="speakers-container">
+				<div class="speaker-image">
+					<img src="./img/hamgurger.png" width="150px" alt="Speaker image">
+				</div>
+				<div class="details">
+					<h2>Keynote Speakers</h2>
+					<p class="speaker-position">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Animi, quo.</p>
```
Comment (aimalamiri) [inline index.html]: It's good to see the data is coming from JavaScript, but don't you think this part of the commented code is not required anymore?

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+}));
+
+// Populating od data
+
+const speakerSection = document.querySelector('.grid');
+let render = '';
+
+const speaker = [
+  {
+    image: './img/sp1.png',
```
Comment (aimalamiri) [inline js/script.js]: In your repository the image name extensions are capital `PNG` but you're trying to access lowercase `png`, which is not visible for others. In my side I can't see the images at all.

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+					<img class="main-events-img" src="./img/network.png" alt="Network" 
+					width="50px"/>
+					<h3>Network</h3>
+					<p>
+						Lorem ipsum, dolor sit amet consectetur adipisicing elit. Vitae..
+					</p>
+				</li>
+			</ul>
+			<a class="ds-program" href="#">SEE THE WHOLE PROGRAM</a>
+			<a class="mb-program" href="#">Join GSASET2023 Submit</a>
```
Comment (aimalamiri) [inline index.html]: Well! don't you think the button should be a little bit smaller and center aligned in horizontal direction? please make sure to fix it.

image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+					delectus nulla. Culpa rerum perspiciatis, quia delectus possimus 
+					eligendi non id nulla asperiores officia quaerat, ab hic sit corporis,
+					reprehenderit minus soluta nam rem ipsum.
+				</p>
+			</div>
+			<p class="p-2">
+				Please contact us via Email for any further questions about the
+				Global Web Development Summit 2023
+			</p>
+			<a class="mail" href="#">info@bhitech.com</a>
```
Comment (aimalamiri) [inline about.html]: Good job! but in here you still need to work on the font-size spacing, and colors a little bit. Kindly make sure to fix it.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @mrbhi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The pages are designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
You did a great job! but you can improve your project by following the original design idea, so please try update some parts of your design as mentioned in the comments below. I'm really surprised that such a big project has only 3 git commits, it's highly recommended to commit your code too often (don't worry you're not supposed to change anything about your git commits 😊), and the last point is that on the video you forget to show a part of your code that makes you feel proud.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 7 above)
Comment (mrbhi) [reply about.html]: Good day, @aimalamiri,
Please kindly help me check through it before submitting it for a second review.
Thanks.

File: css/index.css [hunk 8]
```diff
@@ -0,0 +1,578 @@
...
+  .program {
+    display: flex;
+    list-style: none;
+    gap: 1%;
+  }
+
+  .program li {
+    text-align: center;
+    width: 100%;
+    background-color: #272731ad;
```
Comment (zieeco) [inline css/index.css]: - Your background is close but you can adjust it also to be very close to the design. Here's the color 
`background-color: rgba(255, 255, 255, 0.1);`

Comment (zieeco) [reply css/index.css]: # VIDEO PRESENTATION

- As it's a requirement you should show a part of your code that you're proud of, please, kindly update the video and show some code from your VSCode editor that you're proud of as requested by the previous reviewer as this is a core requirement to have your project approved. see the image below, the part highlighted in a red line below:

Screenshot from 2022-10-27 14-23-14

---

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @mrbhi,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @mrbhi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**